app/ui/browser/views/page/status.jsx

```diff
@@ -51,6 +55,13 @@ Status.displayName = 'Status';
 
 Status.propTypes = {
   page: PropTypes.object.isRequired,
+  statusText: PropTypes.string.isRequired,
 };
 
-export default Status;
+function mapStateToProps(state) {
+  return {
+    statusText: getStatusText(state),
```

Comment:
I wanted to ask about this... since we're only showing one status text at the moment, do we want to keep track of all of them? It seems like we could get away with only storing one since the user will only be hovering over one link at a time. But if we want to show multiple, then we can definitely add a map here.